util/patricia_trie/benches/trie.rs

```diff
@@ -69,9 +74,9 @@ fn trie_insertions_32_mir_1k(b: &mut Bencher) {
 	};
 	let d = st.make();
 	b.iter(&mut ||{
-		let mut memdb = MemoryDB::new();
-		let mut root = H256::new();
-		let mut t = TrieDBMut::new(&mut memdb, &mut root);
+		let mut memdb = MemoryDB::<KeccakHasher>::new();
+		let mut root = <KeccakHasher as Hasher>::Out::new();
```

Comment:
`H256::new()` looks nicer and should still work ;)

Reply:
True. My plan was to make a final "esthetics" pass and fix up things like this once everything is nailed down and fully working.